Assignment2/ruliii-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+	"@context":{
+		"xsd":"http://www.w3.org/2001/XMLSchema#",
+		"ns": "http://www.ontologies.org/ontologies/",
+		"Class01":{
+          "@id":"http://www.ontologies.org/ontologies/Class",
+			"@type":"ns:class"
+		},
+		"Sensor029":{
+          "@id":"http://www.ontologies.org/ontologies/Sensors",
+			"@type":"ns:sensor"
+		},
+		"Measurement8401":{
+          "@id":"http://www.ontologies.org/ontologies/Measurements",
+			"@type":"ns:measuerment"
+		},
+		"temperature":"xsd:long",
+		"date":"xsd:dateTime",
+		"Computer101":{
+          "@id":"http://www.ontologies.org/ontologies/Computers",
+			"@type":"ns:computer"
+		},
+		"User10A":{
+          "@id":"http://www.ontologies.org/ontologies/Users",
+		"@type":"ns:user"
+		},
+		"name":"xsd:string",
+		"includes":"ns:includes",
+		"hasOwner":"ns:hasOwner",
+      "hasMeasurement":"ns:hasMeasuremet"
+	},
+  "Class01":{
+    "includes":{
+  			"Sensor029":{
+              "hasMeasurement":{
+                "Measurement8401":{
+						"temperature":"29",
+						"date":"2010-06-12T12:00:12"
+					}
+              }
+			},
+			"Computer101":{
+              "hasOwner":{
+                "User10A":{
+					"name":"Pedro"
+				}
+              }
+		}
+  		}
+		
+	}
+}
```

Comment:
The current structure of this file generates several blank nodes and wrong relationships between objects. Please reorganize it and try to follow the example we saw in class. 
Remember, when you define something between " " means relationships between objects. Thus, "Class01", "Sensor029", etc. are resources instead of relationships.